test/EFCore.PG.FunctionalTests/Query/RangeQueryNpgsqlFixture.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.TestUtilities;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Npgsql.EntityFrameworkCore.PostgreSQL.TestUtilities;
+using NpgsqlTypes;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query
+{
+    /// <summary>
+    /// Represents a fixture suitable for testing range operators.
+    /// </summary>
+    public class RangeQueryNpgsqlFixture : IDisposable
```

Comment:
In EF Core the standard now seems to be to include the fixture as an inner class of the test suite class. This makes sense since the fixture is only used within its test suite, and this way we have one file per test suite - easier to navigate the ever-growing list of test suites. Let's do this here as well.